src/averageAges.js

```diff
@@ -14,12 +15,32 @@
  *
  * @return {number}
  */
+
+function averageAge(people) {
+  return (
+    people.reduce((years, person) => years + (person.died - person.born), 0) /
+    people.length
+  );
+}
+
+function isChildren(people, motherName) {
```

Comment:
Based on your code, it seems like this function should be named `hasChildren` or `isMother`.